slack_sdk/rtm/v2/__init__.py

```diff
@@ -203,8 +211,11 @@ def connect(self):
 
         old_session: Optional[Connection] = self.current_session
 
+        if self.wss_uri is None:
+            self.wss_uri = self.issue_new_wss_url()
+
         self.current_session = Connection(
-            url=self.issue_new_wss_url(),
```

Comment:
This has been a bug where `connect()` method unnecessarily calls `rtm.connect` API inside the method.